Marquee: fix infinite animation

gwolny opened the pull request:
[Issue] N/A
[Problem] Infinite animation is running only for one iteration.
[Solution] Change "iteration" type to "string". When using string both
number and "infinite" are parsed correctly.

Signed-off-by: Grzegorz Wolny <g.wolny@samsung.com>

Commit:
Marquee: fix infinite animation

[Issue] N/A
[Problem] Infinite animation is running only for one iteration.
[Solution] Change "iteration" type to "string". When using string both
number and "infinite" are parsed correctly.

Signed-off-by: Grzegorz Wolny <g.wolny@samsung.com>

lslachciak reviewed (changes requested):
Besides style errors reported by Travis
I think you should check if parseInt returns NaN
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

Commit:
Fix style error

Issue: N/A
[Problem] Eslint complains about style and undefined "equal" token
[Solution] Correct style in marquee.js and add "equal" to globals

Signed-off-by: Grzegorz Wolny <g.wolny@samsung.com>

gwolny commented:
Before the patch if not a number string was given, it was always converted to the NaN. With the patch string "infinite" is correctly handled. Other are converted to the NaN. So for other cases it works as before.

lslachciak commented:
please wait with merge

TomaszLukawskiSam commented on .eslintrc.js:
```diff
@@ -336,6 +336,8 @@ module.exports = {
 	"globals": {
-		"tizen": true
+		"tizen": true,
+		"equal": false,
```
This is not general true. The "equal" and "test" are only in tests.

Better solution is define
`/* global equal, test */` in specific file
or better modify `/*global test */` and modify test like:
```
	test("API of core Marquee Widget", function (assert) {
             assert.equal( ... );
     ...   
```

TomaszLukawskiSam reviewed (approved):
Looks good to me.
